src/ui/components/ExperimentBrowserPage.tsx

```diff
@@ -0,0 +1,113 @@
+import { FC, useCallback, useEffect, useState } from "react";
+
+const PROD_URL =
+  "https://experimenter.services.mozilla.com/api/v7/experiments/";
+const STAGE_URL =
+  "https://stage.experimenter.nonprod.dataops.mozgcp.net/api/v6/experiments/";
+
+interface Experiment {
+  id: string;
+  userFacingName: string;
+  userFacingDescription: string;
+  channel: string;
+  schemaVersion: string;
+  isEnrollmentPaused: boolean;
+}
+
+const ExperimentBrowserPage: FC = () => {
+  const [environment, setEnvironment] = useState("prod");
+  const [status, setStatus] = useState("Live");
+  const [experiments, setExperiments] = useState<Experiment[]>([]);
+
+  const fetchExperiments = useCallback(
+    async (forceRefresh = false) => {
+      const baseUrl = environment === "prod" ? PROD_URL : STAGE_URL;
+      const refreshString = forceRefresh ? `&bust-cache=${Date.now()}` : "";
+      const url = `${baseUrl}?status=${status}${refreshString}`;
```

Comment:
I would build the URL either with [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) or: 
```
let url = environment === "prod" ? PROD_URL : STAGE_URL;
url += `?status=${status}`
if (forceRefresh) {
  url += `&bust-cache=${Date.now()}`;
}
```